app/views/tags/_grouped_admin_checkboxes.html.haml

```diff
@@ -0,0 +1,20 @@
+%div
+  - i=0
+  - alpha_tags.each_slice(6) do |gr|
```

Comment:
No need to slice, just use a column layout! Using the autoprefixer it's as easy as
```css
.six-columns {
  column-count: 6;
}
```
and put the boxes one under the other.